play/src/main/scala/com/managementapp/database/dao/QuizItemsDAO.scala

```diff
@@ -0,0 +1,27 @@
+package com.managementapp.database.dao
+
+import com.google.inject.Inject
+import com.managementapp.database.models.QuizItems
+import com.managementapp.services.common.ManagementAppDatabase
+import play.api.db.slick.DatabaseConfigProvider
+import slick.driver.MySQLDriver.api._
+
+import scala.concurrent.Future
+
+class QuizItemsTable(tag: Tag) extends Table[QuizItems](tag, "QUIZ_ITEMS") {
```

Comment:
Do we need foreign keys in this table to indicate which QuizSet this QuizItem belongs to?